### PR TITLE
Add installModuleIf with otherwise Module

### PR DESCRIPTION
**Pull request by kokosing**
Add installModuleIf with otherwise Module

### CODE REVIEW

**Comment by kokosing** [comment]
@electrum Would you like to review?

**Comment by electrum** [comment]
Looks good. Please squash and add to `CHANGES`

File: configuration/src/main/java/io/airlift/configuration/ConfigurationModule.java [hunk 1]
```diff
@@ -17,14 +17,29 @@
...
 
 import java.lang.annotation.Annotation;
+import java.util.stream.Stream;
 
 import static io.airlift.configuration.ConfigBinder.configBinder;
 
 public class ConfigurationModule
         implements Module
 {
+    public static ConfigurationAwareModule installModules(Module... modules)
```
Comment (electrum) [inline configuration/src/main/java/io/airlift/configuration/ConfigurationModule.java]: Let's move this below, next to the other static method. We normally only put static factories for the current class at the top. Having methods here distracts from the class fields.

**Comment by kokosing** [comment]
@electrum Comments addressed.

File: CHANGES [hunk 2]
```diff
@@ -1,10 +1,15 @@
+0.180
+
+- Add installModuleIf method to ConditionalModule with ability to install module when condition is not meet
```
Comment (electrum) [inline CHANGES]:
```suggestion
- Add installModuleIf with otherwise Module.
```

File: CHANGES [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+
 0.179
 
 - Update to BVal 2.0.0
 - Report server port when bind fails in HttpServer.
 - Redirect from /resource to /resource/
 - Add future callback overloads that take an executor.
 - Allow configuring the HTTP client Kerberos service principal pattern.
+- Report server port when bind fails in HttpServer
```
Comment (electrum) [inline CHANGES]: Duplicate, likely due to merge conflict
```suggestion
```

File: CHANGES (hunk 2 above)
Comment (electrum) [reply CHANGES]: The original is fine too, please wrap at ~80 characters

**Comment by kokosing** [comment]
Updated.

**Comment by electrum** [comment]
Merged, thanks!

**Comment by kokosing** [comment]
Thanks

**Pull request closed without merging**